project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.typesafe.play" % "interplay"      % "3.1.0-RC7")
+// addSbtPlugin("com.typesafe.play" % "interplay"      % "3.1.0-RC7")
```

Comment:
Like discussed, please keep interplay for now, because it sets up more than just the Scala versions. We will remove interplay at some point, but let's keep the pull request focused on Scala 3 for now.